### PR TITLE
fix: Repair docker compose

### PR DESCRIPTION
**Pull request by luke-biel**

### CODE REVIEW

File: deployment/compose/setup.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  docker-compose -f docker-compose.yml up -d \
+    druid_coordinator druid_broker druid_historical druid_middlemanager druid_router
+fi
+
+sleep 5s
+
+docker-compose \
+  -f docker-compose.yml \
+  up -d \
+  schema_registry # object builder needs it to be running
```
Comment (frondeus) [inline deployment/compose/setup.sh:41]: Object builder uses BB8 pool now, so its no longer true. We don't have to run SR first.

**Comment by frondeus** [comment]
Besides that, LGTM, although it does not unify configuration files with HELM, but I feel it might be a bigger challenge than I thought. We would have to somehow set different `[services]` for HELM (cause different DNS) than for D-C.

**Comment by frondeus** [comment]
One thought - what about bare metal? By removing `.cdl` folder do we say that D-C is the preferred way of running CDL on local machines? Should I maintain my own config files set for Horust/ raw `cargo run`?

**Comment by luke-biel** [comment]
> One thought - what about bare metal? By removing `.cdl` folder do we say that D-C is the preferred way of running CDL on local machines? Should I maintain my own config files set for Horust/ raw `cargo run`?

`.cdl` is not being dropped, in this PR there are only minor tweaks to it. D-C is there only for legacy reasons

File: deployment/compose/setup.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+COMMUNICATION_METHOD=${1:-"kafka"}
+REPOSITORY_KIND=${2:-"postgres"}
+
+QS_TYPE="document"
+
+if [ "$REPOSITORY_KIND" == "victoria_metrics" ] || [ "$REPOSITORY_KIND" == "druid" ]; then
```
Comment (esavier) [inline deployment/compose/setup.sh:9]: druid is non standard, it can be left out

File: deployment/compose/setup/configuration/default.toml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[druid]
+url = "http://router:8888/druid/v2"
+topic = "cdl.timeseries.internal.druid"
+
+[notifications]
+destination = "cdl.reports"
+
+[monitoring]
+metrics_port = "0" # Random
```
Comment (esavier) [inline deployment/compose/setup/configuration/default.toml:36]: i think we cant always leave status ports as random, i.e. i would prefer some specific port over random one, but it have to be different for each service
all deployments can use status endpoints for service monitoring, same with user. Probably also there will be tests for those later so letting those be random doesn't make sense without config service (i.e. place to notify about those ports)

File: deployment/compose/setup.sh (hunk 2 above)
Comment (luke-biel) [reply deployment/compose/setup.sh:9]: I wanted to cover it for as long as we don't delete it

File: deployment/compose/setup/configuration/default.toml (hunk 3 above)
Comment (luke-biel) [reply deployment/compose/setup/configuration/default.toml:36]: Yup, we can't always, but at this point, we weren't doing much with them on docker-compose anyway. If an user wants to turn on metrics/status they can change these in their local workspace

Comment (luke-biel) [reply deployment/compose/setup/configuration/default.toml]: I've made metrics and status a static port in d-c by default

**Comment by frondeus** [comment]
> > One thought - what about bare metal? By removing `.cdl` folder do we say that D-C is the preferred way of running CDL on local machines? Should I maintain my own config files set for Horust/ raw `cargo run`?
> 
> `.cdl` is not being dropped, in this PR there are only minor tweaks to it. D-C is there only for legacy reasons

Sorry, I don't know why I thought you removed whole folder

**Review by frondeus — APPROVED** [review]

**Pull request merged**